Marlin/src/HAL/HAL_STM32F1/persistent_store_flash.cpp

```diff
@@ -43,10 +43,10 @@ namespace HAL {
 namespace PersistentStore {
 
 namespace {
-// Store settings in the last two pages
-// Flash pages must be erased before writing, so keep track.
-bool firstWrite = false;
-uint32_t pageBase = EEPROM_START_ADDRESS;
+  // Store settings in the last two pages
+  // Flash pages must be erased before writing, so keep track.
+  bool firstWrite = false;
+  uint32_t pageBase = EEPROM_START_ADDRESS;
```

Comment:
I thought C++ coding style prescribed not indenting the contents of a `namespace`.
At least they advise so in Google's coding style:
https://google.github.io/styleguide/cppguide.html#Namespaces

Reply:
This is true, namespaces aren't really used in marlin so I don't think there is even a mention on the coding standards page, I'm curious why you found it necessary to hide the variables in an anonymous namespace though, unless they are exposed as externs through the interface they are (pretty much) private to that compilation unit. 

I should probably give up on the idea of introducing Marlin to namespaces and rewrite it as a static class though.

Reply:
Contents _are_ indented in Google's styleguide…
![image](https://user-images.githubusercontent.com/698003/42031355-99be3f90-7a9b-11e8-90db-8b922c6f0c57.png)